resources/RULES.md

```diff
@@ -23,4 +23,4 @@ _ _
 **7.** If your username can't be easily mentioned, please set your nickname to something mentionable. For example, "ᐃᑦᑎᓂᖅᓯᐅᑐᖅ ᑕᖅᓴᖅ" isn't particularly useful.
 
 _ _
-**8.** No political / controversial content on this server. If you wish to discuss those head over to: https://discord.gg/q4KdJFP
+**8.** No political / controversial content on this server. If you wish to discuss those topics, please find another place.
```

Comment:
```suggestion
**8.** Highly controversial content regarding politics, religion and the likes is not allowed.
```

Reply:
That wording is a bit awkward.  Something like `Controversial content, including politics and religion, is not allowed.`?  Keeps the same meaning with fewer, and in my opinion, clearer words.

Reply:
👍